smartos test failure with SIGILL, or perhaps out of memory?

@nodejs/platform-smartos Does anyone have an interpretation of this failure? It says `SIGILL`, which is worrying, but also there are references to out-of-memory.

https://ci.nodejs.org/job/node-test-commit-smartos/nodes=smartos18-64/26258/console

```
12:34:18 Building addon in /home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/test/addons/make-callback-domain-warning
12:34:18 make[2]: Entering directory '/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/test/addons/hello-world-function-export/build'
12:34:18   CXX(target) Release/obj.target/binding/binding.o
12:34:18   SOLINK_MODULE(target) Release/obj.target/binding.node
12:34:18   COPY Release/binding.node
12:34:18 make[2]: Leaving directory '/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/test/addons/hello-world-function-export/build'
12:34:18 gyp info it worked if it ends with ok
12:34:18 gyp info using node-gyp@3.8.0
12:34:18 gyp info using node@13.0.0-pre | sunos | x64
12:34:18 gyp info chdir /home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/test/addons/hello-world-function-export
12:34:18 gyp info spawn /opt/local/bin/python2.7
12:34:18 gyp info spawn args [
12:34:18 gyp info spawn args   '/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/deps/npm/node_modules/node-gyp/gyp/gyp_main.py',
12:34:18 gyp info spawn args   'binding.gyp',
12:34:18 gyp info spawn args   '-f',
12:34:18 gyp info spawn args   'make',
12:34:18 gyp info spawn args   '-I',
12:34:18 gyp info spawn args   '/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/test/addons/hello-world-function-export/build/config.gypi',
12:34:18 gyp info spawn args   '-I',
12:34:18 gyp info spawn args   '/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/deps/npm/node_modules/node-gyp/addon.gypi',
12:34:18 gyp info spawn args   '-I',
12:34:18 gyp info spawn args   '/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/common.gypi',
12:34:18 gyp info spawn args   '-Dlibrary=shared_library',
12:34:18 gyp info spawn args   '-Dvisibility=default',
12:34:18 gyp info spawn args   '-Dnode_root_dir=/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64',
12:34:18 gyp info spawn args   '-Dnode_gyp_dir=/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/deps/npm/node_modules/node-gyp',
12:34:18 gyp info spawn args   '-Dnode_lib_file=/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/$(Configuration)/node.lib',
12:34:18 gyp info spawn args   '-Dmodule_root_dir=/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/test/addons/hello-world-function-export',
12:34:18 gyp info spawn args   '-Dnode_engine=v8',
12:34:18 gyp info spawn args   '--depth=.',
12:34:18 gyp info spawn args   '--no-parallel',
12:34:18 gyp info spawn args   '--generator-output',
12:34:18 gyp info spawn args   'build',
12:34:18 gyp info spawn args   '-Goutput_dir=.'
12:34:18 gyp info spawn args ]
12:34:18 gyp info spawn make
12:34:18 gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build', '--jobs', 4 ]
12:34:18 gyp info ok 
12:34:18 Building addon in /home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/test/addons/make-callback-recurse
12:34:19 /home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/tools/build-addons.js:58
12:34:19 main(process.argv[3]).catch((err) => setImmediate(() => { throw err; }));
12:34:19                                                           ^
12:34:19 
12:34:19 Error: Command failed: /home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/out/Release/node /home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/deps/npm/node_modules/node-gyp/bin/node-gyp.js rebuild --directory=/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/test/addons/make-callback-recurse
12:34:19 gyp info it worked if it ends with ok
12:34:19 gyp info using node-gyp@3.8.0
12:34:19 gyp info using node@13.0.0-pre | sunos | x64
12:34:19 gyp info chdir /home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/test/addons/make-callback-recurse
12:34:19 
12:34:19 
12:34:19 #
12:34:19 # Fatal error in , line 0
12:34:19 # Fatal process out of memory: Zone
12:34:19 #
12:34:19 #
12:34:19 #
12:34:19 #FailureMessage Object: fffffbffee6094d0
12:34:19     at ChildProcess.exithandler (child_process.js:290:12)
12:34:19     at ChildProcess.emit (events.js:200:13)
12:34:19     at maybeClose (internal/child_process.js:1028:16)
12:34:19     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5) {
12:34:19   killed: false,
12:34:19   code: null,
12:34:19   signal: 'SIGILL',
12:34:19   cmd: '/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/out/Release/node ' +
12:34:19     '/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/deps/npm/node_modules/node-gyp/bin/node-gyp.js ' +
12:34:19     'rebuild ' +
12:34:19     '--directory=/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/test/addons/make-callback-recurse',
12:34:19   stdout: '',
12:34:19   stderr: 'gyp info it worked if it ends with ok\ngyp info using node-gyp@3.8.0\n' +
12:34:19     'gyp info using node@13.0.0-pre | sunos | x64\ngyp info chdir ' +
12:34:19     '/home/iojs/build/workspace/node-test-commit-smartos/nodes/smartos18-64/test/addons/make-callback-recurse\n' +
12:34:19     '\n\n#\n# Fatal error in , line 0\n# Fatal process out of memory: Zone\n#\n#\n' +
12:34:19     '#\n#FailureMessage Object: fffffbffee6094d0'
12:34:19 }
12:34:19 make[1]: *** [Makefile:402: test/addons/.buildstamp] Error 1
```

--- a/ansible/roles/jenkins-worker/templates/jenkins_manifest.xml.j2
+++ b/ansible/roles/jenkins-worker/templates/jenkins_manifest.xml.j2
@@ -21,7 +21,7 @@
           <envvar name='NODE_TEST_DIR' value='/home/{{ server_user }}/tmp' />
           <envvar name='OSTYPE' value='solaris' />
           <envvar name='HOME' value='/home/{{ server_user }}' />
-          <envvar name='JOBS' value='{{ jobs_env|default("4") }}' />
+          <envvar name='JOBS' value='{{ jobs_env|default("2") }}' />
           <envvar name='PATH' value='/usr/local/bin:/usr/local/sbin:/opt/local/sbin:/opt/local/bin:/usr/sbin:/usr/bin:/sbin' />
         </method_environment>
       </method_context>

--- a/ansible/roles/jenkins-worker/vars/main.yml
+++ b/ansible/roles/jenkins-worker/vars/main.yml
@@ -144,5 +144,5 @@ jobs_variants: {
   armv6l: '1',
   armv7l: '2',
   arm64: '3',
-  smartos: '4'
+  smartos: '2'
 }

